Source/FullScreenPokemon.ts

```diff
@@ -9384,6 +9385,7 @@ module FullScreenPokemon {
          */
         checkForOldStorageData(): void {
             if (!this.ItemsHolder.getItem("oldLocalStorage") || this.ItemsHolder.getItem("gameStarted")) {
+                this.ItemsHolder.getItem("collectionKeys");
```

Comment:
This is for when the user starts a new game with data in localStorage (so 'CONTINUE' is an option). Nothing specifically references `collectionKeys` at the start  and brings the data from oldLocalStorage to `ItemsHolder.items` so `["Blank"]` will be the value of `collectionKeys` when 'NEW GAME' is selected.